terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -105,12 +103,22 @@ public void test01_03_redirectToExternalLink() {
         // screen capture
         screenCapture.save(driver);
 
+        ApServerName apServerName = webDriverOperations.getApServerName();
+        String apServerVersion = webDriverOperations.getApServerVersion();

```

Comment:
9ef5d85566dac5eed657b7b82a9a4aea536eef32 
I shift the place getting application server information at first of test case at above commit.